packages/frontend-2/components/workspaces/Promo/Page.vue

```diff
@@ -0,0 +1,73 @@
+<template>
```

Comment:
I know we do this in other pages as well, but is there any specific reason why we have a separate component for pages, instead of use just using the page .vue file?

Reply:
The only reason I did in in this branch is so we can move this easily. Currently, it lives at /workspaces/, but in future this will be a list of workspaces